src/main/resources/templates/course.twig

```diff
@@ -13,27 +13,36 @@
         <div class="course-editor-grid">
             <div id="course-seating-grid"></div>
         </div>
-        <div class="course-editor-controls">
-            <button id="editsave" type="button" class="btn btn-primary">Edit</button>
-        </div>
+        {% block buttons %}
+        {% endblock %}
     </div>
 {% endblock %}
 
 {% block script %}
     {{ parent() }}
     <script type="text/javascript" src="/static/js/grid.js"></script>
     <script>
+        var user = {
+            id: {{ user.id }},
+            username: '{{ user.username }}'
+        }
+        console.log(user);
```

Comment:
I noticed there is a fair amount of console logging. I think it might be a good idea to limit what we display and how often we display information to the console. For me, when I am debugging javascript, I like to use logging to display values and make sure my code works correctly, but it's a bit difficult to do when there's lots in the console window.

Reply:
Yeah I'll remove this and refactor the JS a bit